SyntheticsRunTestsTask/jest.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   clearMocks: true,
-  moduleFileExtensions: ['js', 'ts'],
+  moduleFileExtensions: ['js', 'ts', 'node'],
```

Comment:
This addition solves [this issue](https://github.com/DataDog/datadog-ci-azure-devops/actions/runs/6864590747/job/18666703238#step:6:10).